src/app/external-cluster-wizard/steps/external-cluster/provider/eks/component.ts

```diff
@@ -197,6 +216,27 @@ export class EKSClusterSettingsComponent
     this.selectedNodeRolArn = this.nodeRoles.find((nodeRole: EKSNodeRole) => nodeRole.roleName === roleName)?.arn;
   }
 
+  onInstanceTypeChange(instanceType: string[]): void {
+    if (instanceType) {
+      this.selectedInstanceTypes = instanceType;
+    } else {
+      this.selectedInstanceTypes = [];
+    }
+  }
+
+  instanceDisplayName(): string[] {
```

Comment:
I see some issue in this function IMO it should be getting value from inside km-combobox -> as values[] 

Issues:

https://user-images.githubusercontent.com/17727069/192577873-4581c0a0-ee40-4be6-a745-c326775bf367.mp4

https://user-images.githubusercontent.com/17727069/192577921-8ce98002-abe9-4403-adf2-1f369fd04dfc.mp4


How I tried to fixed it:
```suggestion
  instanceDisplayName(values: string[]): string[] {
    return values
      ? values.map((item: string) => {
        const selectedObj = this.instanceTypes.find((instanceObj: EKSInstanceTypeList) => instanceObj.name === item);
        return selectedObj && `${selectedObj.pretty_name} (${selectedObj.vcpus} vCPUs, ${selectedObj.memory} GB RAM)`;
      })
      : [];
  }
```

https://user-images.githubusercontent.com/17727069/192578258-e4598bcd-fa5f-48a8-a6bf-de0c37b2ee8c.mp4